global.json

```diff
@@ -1,5 +1,7 @@
 {
   "sdk": {
+    "version": "7.0.100",
+    "rollForward": "major",
```

Comment:
Suggest: `latestMajor` here, which will treat 7.0.100 as a minimum but continue working into the 8.x SDK as 7.x ages out into the next LTS. It _looks_ like you're only intending a minimum for support level, so if that's the case, just less to do later with latestMajor!

Reply:
👍

Reply:
Yeah, all I really want is for the lang-ver demands to be obviously enforced by SDK level, to give a meaningful error message as soon as possible in the build process

Reply:
@NickCraver tweaked, ty sir